smoke-tests/src/test/java/com/splunk/opentelemetry/WebLogicSmokeTest.java

```diff
@@ -54,15 +57,29 @@ public void webLogicSmokeTest(String imageName) throws IOException, InterruptedE
     // FIXME: APMI-1300
     //    assertServerHandler(....)
```

Comment:
You can replace this FIXME with just a comment, that there aren't any server handlers for weblogic because of APMI-1300.